src/aosm/azext_aosm/cli_handlers/onboarding_vnf_handler.py

```diff
@@ -177,70 +54,35 @@ def build_artifact_list(self):
             Path(VNF_OUTPUT_FOLDER_FILENAME, ARTIFACT_LIST_FILENAME), artifact_list
         )
 
-    def build_resource_bicep(self):
+    def build_resource_bicep(self) -> BicepDefinitionElementBuilder:
         """Build the resource bicep file."""

```

Comment:
Could we get some better doc strings? First line in fine. But could you add another paragraph saying what the bicep file is used for, and what it's built from, and/or an example.